### ISSUE TITLE
Alert component not accepting html tags in html property

### ISSUE THREAD

**Issue by Mohanrau** [quasarframework/quasar#831]
### Software version

Quasar:0.14.1
OS:windows 10
Node: 5.^
NPM:6.^
Browsers:Chrome
iOS:
Android:
Any other software related to your bug:

### What did you get as the error?
Alert component HTML property not accepting html tags

### What were you expecting?
As per property name it shut accept html tags as value

### What steps did you take, to get the error?
``` 
import { Alert } from 'quasar'
Alert.create({
          enter: 'bounceInRight',
          leave: 'bounceOutRight',
          color: 'positive',
          icon: 'info',
          html: `<h6>` + title + `</h6><br><br>` + message,
          position: 'top-right'
        })
```
Preview
![capture](https://user-images.githubusercontent.com/1044660/29683627-94110678-8941-11e7-9004-f0a32699ab0c.PNG)

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.2

**Comment by rogeriomq** [comment]
I had opened this ISSUE( #731 ).
  Hahaha I knew I was not crazy!
